apps/auction2/client.js

```diff
@@ -22,3 +37,69 @@ if (sd.AUCTION && sd.AUCTION.is_live_open == false) {
   })
   activeBids.start()
 }
+
+// Commercial filtering
+const params = new Params({sale_id: sd.AUCTION.id}, {
+  fullyQualifiedLocations: fullyQualifiedLocations
+})
+const filter = new Filter({params: params})
+
+// Main Artworks view
+filter.artworks.on('reset', () => {
+  const artworkView = new ArtworkColumnsView({
+    collection: filter.artworks,
+    el: $('.cf-artworks'),
+    allowDuplicates: true,
+    gutterWidth: 30,
+    numberOfColumns: 3,
+    context_page: 'Collect page'
+  })
+})
+
+// Header
+const totalView = new TotalView({
+  el: $('.cf-total-sort__total'),
+  filter: filter,
+  artworks: filter.artworks
+})
+
+const sortView = new SortView({
+  el: $('.cf-total-sort__sort'),
+  params: params
+})
+
+// Sidebar
+const mediumFilterView = new MediumFilterView({
```

Comment:
It might make sense to compose all of these views into one container view. Generally it's good to keep these "index" client-side files lean because they tend to have a fair bit of "glue" code that doesn't lend itself well to testing. This looks like a pretty straightforward list of views, but it still might help separate the parts that are the view hiearchy bit from the "glue", e.g. you could collapse all these as sub-views of an `AuctionPage` and this init code could be something like...

````javascript
import ...

// Models
const auction = new Auction(...)
const params = new Params(...)
//...

// Views
new AuctionPage({ params, auction, ... })

// Init
Backbone.history.start({pushState: true})
params.trigger('change')
````